lib/v2/devolverdigital/blog.js

```diff
@@ -0,0 +1,43 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+
+module.exports = async (ctx) => {
+    const baseUrl = 'https://www.devolverdigital.com/blog';
+
+    const { data: response } = await got(baseUrl);
+    const $ = cheerio.load(response);
+
+    const nextData = JSON.parse($('#__NEXT_DATA__').html());
+    const allBlogContents = $('div.flex > div > div > div > div:not([class])');
+
+    const items = nextData.props.pageProps.posts.map((post, postIndex) => {
+        // img resource redirection and
+        // clean up absolute layouts for img and span
+        const imageUrls = [];
+        post.body.forEach((item) => {
+            if (item.type === 'upload' && item.value.cloudinary.resource_type === 'image') {
+                imageUrls.push(item.value.cloudinary.secure_url);
+            }
+        });
+        const allImageSpans = $(allBlogContents[postIndex]).find('span > img').parent();
+        allImageSpans.each((spanIndex, span) => {
+            $(span).attr('style', $(span).attr('style').replace('position:absolute'), '');
```

Comment:
[.attr](https://cheerio.js.org/docs/api/classes/Cheerio#attr) does not take more than 2 parameters